cfbs/pretty.py

```diff
@@ -233,10 +235,11 @@ def _should_wrap(parent):
         return count >= 2
 
     def _encode_list(lst, indent, cursor):
-        if not lst:
+        if not lst and indent == 0:
+            return "[\n]"
+        elif not lst:
             return "[]"
```

Comment:
@mineralsfree @larsewi How about keeping it on one line if it's completely empty?

Reply:
I agree :+1: